pybossa/api/task.py

```diff
@@ -70,7 +70,7 @@ def _update_attribute(self, new, old):
             if not gold_task and (n_taskruns >= new.n_answers):
                 new.state = 'completed'
         new.calibration = int(gold_task)
-        new.exported = gold_task
+        new.exported = gold_task or new.exported
```

Comment:
Yes, this is a good fix. This is why I could not use the API call. Also, we had discussed this very issue before, if you call, last year. See https://github.com/bloomberg/pybossa/pull/495/files
:)